docs/source/content/deep_learning/multilayer.rst

```diff
@@ -0,0 +1,151 @@
+
+.. _proven: https://en.wikipedia.org/wiki/Universal_approximation_theorem
+
+***************
+What is a node?
+***************
+
+A node is a single unit in a neural network.
+Nodes **activate** at different levels depending on a weighted sum of the previous layer's nodes.
+In practice, the actual activation is the result of a **sigmoid function**
+applied to this result, but we will skip over that detail here for simplicity.
+Thinking of the output in this way won't lose any of the magic of neural networks, while avoiding some painful details.
+In MLPs, nodes activate based on **all** of the nodes in the previous layer.
+
+
+In this example, we let's focus on the single-node layer, which is that way for example purposes.
+Each line represents the weights of the nodes in the previous layer.
+The sum of the weights of each connection multiplied by the activation of the
+connected node results in the activation of our node.
+The key here is the weights, since they determine the output of the node.
+Remember that nodes only take input from the previous layer, so the weights are
+the only differentiator of nodes in the same layer.
+
+*************************************
+What defines a multilayer perceptron?
+*************************************
+
+A Multilayer Perceptron (MLP) is a type of **feedforward** neural network which is characterized
+by an input layer, some number of intermediate layers, and an output layer, which are **fully connected**.
+An MLP uses **backpropagation** for training.
+The term **feedforward** refers to the layered architecture in the network,
+specifically that there are no cycles in the network.
+The layer structure ensures no cycles exists, as layers are only allowed to have
+weights from the directly previous layer.
+The term **fully connected** refers to the fact that in MLP, all nodes in a given
+layer have a weight to all of the nodes in the previous layer.
+
+
+************************
+What is backpropagation?
+************************
+
+When training a neural network, the expected output is a level of activation for each node in the output layer.
+From that information and the actual activation, we can find the *cost* at each node, and adjust the weights accordingly.
+The idea of backpropagation is to adjust the weights that determine each node's activation based on the cost.
+
+To keep the idea here high-level, we will avoid the details on how the exact math works, and focus on the big picture.
+If you would like to see the math, check out this article_.
+
+.. _article: http://neuralnetworksanddeeplearning.com/chap2.html
+
+Take a look at this screenshot taken from the tensorflow_ test site linked earlier.
+Here, we are training a neural network to classify the blue dots and the orange dots.
+The choices made for the nodes here are arbitrary, and we encourage you to mess around with them.
+
+.. _tensorflow: https://playground.tensorflow.org/
+
+.. figure: _img/MLP_0.PNG
```

Comment:
All of the figures are not showing up

Reply:
fixed that, needed double colon instead of single.